tests/testthat/test-corridor.R

```diff
@@ -1,7 +1,7 @@
-test_that("proper parameters must be provided depending on selected method", {
-  skip_if_not_installed("CRiSpData")
+bucharest_osm <- get_osm_example_data()
```

Comment:
Should we put this and the analogous:
```
bucharest_dem <- get_dem_example_data()
```
In an test helper function (e.g. in `/tests/testthat/helper-example-data.R` ) ? 
This way the datasets should become available to all tests without re-downloading them in all modules.